weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/EditableCallName.tsx

```diff
@@ -0,0 +1,57 @@
+import EditableField from '@wandb/weave/common/components/EditableField';
+import React, {useCallback} from 'react';
+import {useEffect, useState} from 'react';
+
+import {useWFHooks} from '../wfReactInterface/context';
+import {CallSchema} from '../wfReactInterface/wfDataModelHooksInterface';
+import {opNiceName} from './Links';
+
+export const EditableCallName: React.FC<{
+  call: CallSchema;
+}> = ({call}) => {
+  const defaultDisplayName = opNiceName(call.spanName);
+  const setDisplayNameIsEmpty =
```

Comment:
This seems like a confusing name, how about just `isDisplayNameEmpty`

Reply:
fixed